modules/mobile/app/controllers/mobile/v1/immunizations_controller.rb

```diff
@@ -27,11 +25,26 @@ def service
       end
 
       def pagination_params
-        Mobile::V0::Contracts::GetPaginatedList.new.call(
+        @pagination_params ||= Mobile::V0::Contracts::GetPaginatedList.new.call(
           page_number: params.dig(:page, :number),
-          page_size: params.dig(:page, :size)
+          page_size: params.dig(:page, :size),
+          use_cache: params[:useCache] || true
```

Comment:
Unclear if we really need to add this param rather than just always using the cache, but it might be useful for debugging.

Reply:
Same as above.